### PR TITLE
Fix some lingering pagination issues

### PR DESCRIPTION
**Pull request by Scotchester**
Following some discussions with designers and FEWDs, this PR tidies up some issues with cf-pagination

## Changes

- Cleans up styles to ensure pagination is styled as designed at all breakpoints

## Removals

- Removes unneeded variables

## Testing

- _To be updated after forthcoming PR to capital-framework-sandbox_

## Screenshots

screen shot 2017-03-30 at 16 52 20

## Todos

- [x] Update cf-pagination's `usage.md` file (in this PR) to ensure latest recommended markup is being shown
- [ ] Open PR to update capital-framework-sandbox
- [ ] Open PR to update capital-framework's docs site
- [ ] Once this update is released, update cfpb/design-manual#456 to use this code

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-pagination/src/cf-pagination.less [hunk 1]
```diff
@@ -16,10 +16,8 @@
 
 // Sizing variables
 
-@pagination-font-size-px:  @base-font-size-px;
-@pagination-font-size-em:  unit( @pagination-font-size-px / @base-font-size-px, em );
-@pagination-btn-height-px: 42px;
-@pagination-btn-width-px:  130px;
+@pagination-font-size-px:     @base-font-size-px;
```
Comment (jimmynotjim) [inline src/cf-pagination/src/cf-pagination.less]: This is only ever used when calculating layout dimensions, not for setting type sizing. Could we drop it in favor of using `@base-font-size-px` on it's own?

Comment (Scotchester) [inline src/cf-pagination/src/cf-pagination.less]: Cool with me :+1:

Comment (Scotchester) [inline src/cf-pagination/src/cf-pagination.less]: Fixed in https://github.com/cfpb/capital-framework/pull/463/commits/0b66aca860b6ce56b19bb5f0a6e0633ca63d7eb5

**Comment by Scotchester** [comment]
Updated the usage file and this is now ready for final review.

**Comment by jimmynotjim** [comment]
Looks good, merge when you're ready.

**Pull request merged**